src/shared/utils.ts

```diff
@@ -1441,3 +1442,9 @@ export function selectableLanguages(
     }
   }
 }
+
+export function uploadImage(image: File): Promise<UploadImageResponse> {
+  const client = new LemmyHttp(httpBase);
```

Comment:
It looks like there are multiple functions that rely on a lemmy HTTP client newed up with the just the HTTP base URL (i.e. this function, `fetchCommunities`, and `fetchUsers`). It may make more sense to just make a module level variable variable of the HTTP client instead of newing it up every time one of these functions is called. Thoughts?

Reply:
I'd def be good with that. Maybe as a service-y type singleton pattern alongside `WebSocketService` and `UserService`.

The isometric back-end does have to have its own separate http client tho I think, that's in `server/index.ts`

But ya it doesn't have to be a part of this PR, but feel free to add an issue for it.